prepare_iso/prepare_pvm.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh -e
+
+usage() {
+	cat <<EOF
+Usage:
+$(basename "$0") "/path/to/diskimage.vhd"
+
+Description:
+Creates and exports a Parallels virtual machine (PVM) from a virtual disk image
+
+EOF
+}
+
+cleanup() {
+  if [ -n "$VM" ] && prlctl list --all | grep -q "$VM"; then
+    prlctl unregister "$VM" > /dev/null
+  fi
+}
+
+trap cleanup EXIT INT TERM
+
+msg_status() {
+	echo "\033[0;32m-- $1\033[0m"
+}
+
+msg_error() {
+	echo "\033[0;31m-- $1\033[0m"
+}
+
+render_template() {
+  eval "echo \"$(cat "$1")\""
+}
+
+if [ ! -f "$1" ]; then
+  usage
+	exit 1
+fi
+
+HARDDRIVE="$1"
+VM="$(basename "${HARDDRIVE%.vhd}")"
+
+OUTPUT="${HARDDRIVE%.vhd}.pvm"
+ABS_PATH="$(realpath ${OUTPUT})"
```

Comment:
My 10.13.2 system doesn't have a `realpath` (and it seems there isn't a drop-in solution in homebrew-core either):

```
➜  packer git:(1fcc68c) ../prepare_iso/prepare_pvm.sh out/macOS_10.13.2.vhd
../prepare_iso/prepare_pvm.sh: line 43: realpath: command not found
```

Could we replace this with some other Darwin/BSD-compatible code which does the same thing? I'm fine if that means requiring `bash` instead of `sh`.

Reply:
Ah, good catch. It's part of  [`coreutils`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/coreutils.rb).